### ISSUE TITLE
check_network: wrong value returned for sent and total counters

### ISSUE THREAD

**Issue by fabibio** [mickem/nscp#413]
check_network returns received counter value instead of sent and total.

The bug is at lines 118 and 119 of check_network.hpp, where BytesReceivedPersec is returned even for getBytesSentPersec()  and getBytesTotalPersec() :

```
117: long long getBytesReceivedPersec() const { return BytesReceivedPersec; }
118: long long getBytesSentPersec() const { return BytesReceivedPersec; }
119: long long getBytesTotalPersec() const { return BytesReceivedPersec; }
```

Best regards

Fabio

**Comment by mickem** [comment]
Thanks will fix...

**Comment by nmat** [comment]
is this issue still in 0.5.1.44?

**Comment by mickem** [comment]
Yes, but should be fixed in the next 0.5.2 build... please reopen if you want me to back port this...
